src/frontend/qt_sdl/Platform.cpp

```diff
@@ -411,5 +414,14 @@ int LAN_RecvPacket(u8* data)
         return LAN_Socket::RecvPacket(data);
 }
 
+void StartRumble()
+{
+    Input::StartRumble();
```

Comment:
Since those are one-line functions now, `Input::StartRumble()` and `Input::StopRumble()` can be called directly and these intermediaries removed.